socorro/tests/external/es/test_crashstorage.py

```diff
@@ -250,13 +234,12 @@ def test_crash_size_capture(self):
         crashstorage = self.build_crashstorage()
         with MetricsMock() as mm:
             crashstorage.save_processed_crash(
-                raw_crash=raw_crash,
+                raw_crash={},
                 processed_crash=processed_crash,
             )
 
-            mm.assert_histogram("processor.es.raw_crash_size", value=2)
             mm.assert_histogram("processor.es.processed_crash_size", value=96)
-            mm.assert_histogram("processor.es.crash_document_size", value=186)
+            mm.assert_histogram("processor.es.crash_document_size", value=169)
```

Comment:
This value changes because the document no longer contains a `"raw_crash":{},`.

Reply:
Now that I'm thinking about this, we could stop emitting the processed crash size. The "Socorro prod app metrics" dashboard only has a panel for the crash document size. 

I'll fix this now--one less metric.